### PR TITLE
Calculator component

### PR DESCRIPTION
**Pull request by vanelnw**
Today in the math-magician project, we add logic to make basic ccalculation

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,3 +1,8 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable no-plusplus */
+/* eslint-disable no-loop-func */
+/* eslint-disable spaced-comment */
+/* eslint-disable react/no-unused-state */
 /* eslint-disable react/button-has-type */
```
Comment (omar-muhamad) [inline src/components/Calculator.js]: - [ ] Please remove all the `eslint `disablement lines and fix eslint error in your code.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -6,45 +11,58 @@ class Calculator extends Component {
   // eslint-disable-next-line no-useless-constructor
   constructor(props) {
```
Comment (omar-muhamad) [inline src/components/Calculator.js]: - [ ] Same here, Please remove all the `eslint `disablement lines and fix eslint error in your code.

File: src/components/Calculator.js [hunk 3]
```diff
@@ -6,45 +11,58 @@ class Calculator extends Component {
...
-    createDigits = () => {
-      const digits = [];
-      let iterate = 9;
-      while (iterate > 0) {
-        digits.push(<button>{iterate}</button>);
-        iterate -= 1;
-      }
-      return digits;
+  createDigits = () => {
+    // eslint-disable-next-line react/prop-types
```
Comment (omar-muhamad) [inline src/components/Calculator.js]: - [ ] Same here, Please remove all the `eslint `disablement lines and fix eslint error in your code.

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @vanelnw,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @vanelnw 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git-Flow.
✔️ Professional README  
✔️ Good Commit Messages


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 3 above)
Comment (Abenezer-Tilahun) [reply src/components/Calculator.js]: Fixed.

File: src/components/Calculator.js (hunk 2 above)
Comment (Abenezer-Tilahun) [reply src/components/Calculator.js]: Fixed.

File: src/components/Calculator.js (hunk 1 above)
Comment (Abenezer-Tilahun) [reply src/components/Calculator.js]: Fixed.

**Pull request merged**